.github/workflows/ci.yml

```diff
@@ -0,0 +1,194 @@
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.global-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-${{ matrix.node-version }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-yarn-
+    - name: Cache node_modules
+      id: cache-node-modules
+      uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-
+    - name: Install Dependencies
+      run: yarn install --frozen-lockfile
+      if: |
+        steps.cache-global-package-manager-cache.outputs.cache-hit != 'true' ||
+        steps.cache-node-modules.outputs.cache-hit != 'true'
+    - name: Test
+      run: yarn test:ember --launch ${{ matrix.browser }}
+
+  try-scenarios:
+    name: Tests - ${{ matrix.ember-try-scenario }}
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.allow-failure }}
+    needs: test
+
+    strategy:
+      fail-fast: true
+      matrix:
+        ember-try-scenario: [
+          ember-lts-3.4,
+          ember-lts-3.8,
+          ember-lts-3.12,
+          ember-lts-3.16,
+          ember-lts-3.20,
+          ember-release,
+          ember-beta,
+          ember-default-with-jquery,
+          ember-classic
+        ]
+        allow-failure: [false]
+        include:
+          - ember-try-scenario: ember-canary
+            allow-failure: true
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - uses: actions/setup-node@v2-beta
+      with:
+        node-version: '10.x'
+    - name: Get package manager's global cache path
+      id: global-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - name: Cache Yarn cache
+      id: cache-global-package-manager-cache
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.global-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-${{ matrix.node-version }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-yarn-
+    - name: Cache node_modules
+      id: cache-node-modules
+      uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-
+    - name: Install Dependencies
+      run: yarn install --frozen-lockfile
+      if: |
+        steps.cache-global-package-manager-cache.outputs.cache-hit != 'true' ||
+        steps.cache-node-modules.outputs.cache-hit != 'true'
+    - name: Test
+      env:
+        EMBER_TRY_SCENARIO: ${{ matrix.ember-try-scenario }}
+      run: node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO
```

Comment:
I think it's a bit unfortunate that the default `ember-try` script doesn't say `ember try:one`, but `ember try:each`. If your script can modify `package.json`, I think I'd prefer seeing,

```yaml
run: yarn test:ember-compatibility ${{ matrix.ember-try-scenario }}
```

- https://gist.github.com/ijlee2/25004d700cd008ee08ecc4873c109a5b#file-ci-yml-L18
- https://gist.github.com/ijlee2/25004d700cd008ee08ecc4873c109a5b#file-ci-yml-L233

Reply:
Fully agree that a shorthand for `ember try:one` is missing. But `test:ember-compatibility` is already used in recent Ember CLI blueprints as a shortcut for `ember try:each`: https://github.com/ember-cli/ember-addon-output/blob/9534f509f99c366070ed2683a5d84dd2510acffb/package.json#L23 `test:ember-scenario` is an alternative name that I have used in some projects.
 
Technically the `create-github-actions-for-ember-addon` script could modify the `package.json`. But I'm not sure if it should. That might cause confusion and make upgrading from one Ember CLI version to another more difficult.

I think proposing to add an alias for `ember try:one` to default blueprint would be a better solution.